tests/test_client_ws.py

```diff
@@ -54,6 +56,94 @@ async def test_ws_connect(ws_key: Any, loop: Any, key_data: Any) -> None:
     assert hdrs.ORIGIN not in m_req.call_args[1]["headers"]
 
 
+async def test_ws_connect_read_timeout_is_reset_to_inf(
+    ws_key: Any, loop: Any, key_data: Any
+) -> None:
+    resp = mock.Mock()
+    resp.status = 101
+    resp.headers = {
+        hdrs.UPGRADE: "websocket",
+        hdrs.CONNECTION: "upgrade",
+        hdrs.SEC_WEBSOCKET_ACCEPT: ws_key,
+        hdrs.SEC_WEBSOCKET_PROTOCOL: "chat",
+    }
+    resp.connection.protocol.read_timeout = 0.5
+    with mock.patch("aiohttp.client.os") as m_os:
+        with mock.patch("aiohttp.client.ClientSession.request") as m_req:
```

Comment:
I think you probably copied this from the above test, but please combine with statements

Reply:
I can do this, but I'm trying to keep uniformity between master and 3.9 patches. This is the state of affairs in the master. With your permissions I'd like not to rework the entire `test_client_ws` suite - all tests there are like that.

Reply:
Its fine to leave the existing tests as-is. Ideally we don't create any new ones with that pattern though.

Reply:
Sure, I'll fix this.